tck/src/main/java/org/reactivestreams/tck/IdentityProcessorVerification.java

```diff
@@ -387,23 +388,29 @@ public void required_spec104_mustCallOnErrorOnAllItsSubscribersIfItEncountersANo
     optionalMultipleSubscribersTest(2, new Function<Long,TestSetup>() {
       @Override
       public TestSetup apply(Long aLong) throws Throwable {
-        return new TestSetup(env, processorBufferSize) {{
+        return new TestSetup(env, processorBufferSize, false) {{
           final ManualSubscriberWithErrorCollection<T> sub1 = new ManualSubscriberWithErrorCollection<T>(env);
-          env.subscribe(processor, sub1);
-
           final ManualSubscriberWithErrorCollection<T> sub2 = new ManualSubscriberWithErrorCollection<T>(env);
+
+          // connect upstream
+          env.subscribe(this, processor);
+          // connect downstreams
+          env.subscribe(processor, sub1);
+          sub1.request(2);
           env.subscribe(processor, sub2);
+          sub2.request(1);
 
-          sub1.request(1);
+          // request bubbles up to upstream publisher:
           expectRequest();
           final T x = sendNextTFromUpstream();
           expectNextElement(sub1, x);
           sub1.request(1);
 
           // sub1 has received one element, and has one demand pending
-          // sub2 has not yet requested anything
+          // sub2 has received one element, and no more pending demand
```

Comment:
The change is that:
- we did not request from sub2, which made it impossible for "lazy, wait for both" processors to start requesting from upstream
- now we request at least one element from each downstream, causing one element to be signalled down
- we now check that the error signal overtakes the outstanding elements, which was the goal of the test